src/tools/cephfs/top/cephfs-top

```diff
@@ -253,19 +273,23 @@ class FSTop(object):
             return False
         client_metadata = stats_json[CLIENT_METADATA_KEY]
         num_clients = len(client_metadata)
-        num_mounts = len([client for client, metadata in client_metadata.items() if not
-                          metadata[CLIENT_METADATA_MOUNT_POINT_KEY] == 'N/A'])
+        num_mounts = len([client for client, metadata in client_metadata.items() if
+                          CLIENT_METADATA_MOUNT_POINT_KEY in metadata and
+                          metadata.get(CLIENT_METADATA_MOUNT_POINT_KEY) != 'N/A'])
+        num_unknowns = len([client for client, metadata in client_metadata.items() if
+                            metadata.get(CLIENT_METADATA_MOUNT_POINT_KEY) is None])
         num_kclients = len([client for client, metadata in client_metadata.items() if
                             "kernel_version" in metadata])
-        num_libs = num_clients - (num_mounts + num_kclients)
+        num_libs = num_clients - (num_mounts + num_kclients + num_unknowns)
         now = datetime.now().ctime()
         self.header.addstr(0, 0,
                            FS_TOP_VERSION_HEADER_FMT.format(prog_name=FS_TOP_PROG_STR, now=now),
                            curses.A_STANDOUT | curses.A_BOLD)
         self.header.addstr(1, 0, FS_TOP_CLIENT_HEADER_FMT.format(num_clients=num_clients,
```

Comment:
you need to include {num_unknowns} in `FS_TOP_CLIENT_HEADER_FMT`.

BTW, I think we should just drop this for now. I.e., do not track unknown clients. (Yes, I did comment on adding that, but in the interest of time, let's not include this).

Which means this should just be:

```
num_mounts = len([client for client, metadata in client_metadata.items() if CLIENT_METADATA_MOUNT_POINT_KEY in metadata and metadata[CLIENT_METADATA_MOUNT_POINT_KEY] != 'N/A'])
```